docs/en/changelog.md

```diff
@@ -10,7 +10,7 @@
 #### New Features
 
 - Update performance of ReDet on HRSC2016. (#203)
-- Upgrage visualization to custom colors of different classes (#187)
+- Upgrage visualization to custom colors of different classes and bump min mmdet to 2.22.0 (#187, #267, #270)
```

Comment:
Upgrage visualization to custom colors of different classes. This requires mmdet>2.22.0. (#187, #267, #270)